### DISCUSSION TITLE
Get items fom the Endless Replayability Fest 2024

### DISCUSSION THREAD

**Discussion by woctezuma** [chr233/ASFEnhance#258] [杂谈 / General]
![Picture](https://clan.cloudflare.steamstatic.com/images/41316928/ddb35854c30668a338522c6d6a25fbc1c09dd603.jpg)

- https://store.steampowered.com/category/endlessly_replayable
- https://store.steampowered.com/points/shop/event/sale_replayability
- https://store.steampowered.com/points/shop/app/2855120
- https://www.steamcardexchange.net/index.php?gamepage-appid-2855120
- https://steamdb.info/app/2855120/communityitems/

Command:
```sh
!rpi ASF 286022,286023,286024
```

**Comment by botka4aet** [comment]
`!rpi ASF 286022,286023,286024`

**Reply by woctezuma** [reply]
Thank you!

**Comment by chr233** [comment]
thank yoy
